frontend/src/hooks/pims-api/useApiTenants.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useApi, IApiTenants, ITenantConfig } from '.';
+
+/**
+ * PIMS API wrapper to centralize all AJAX requests to the tenants endpoints.
+ * @returns Object containing functions to make requests to the PIMS API.
+ */
+export const useApiTenants = () => {
+  const api = useApi();
+
+  return React.useMemo(
+    () =>
+      ({
+        getSettings: () => api.get<ITenantConfig>('tenants'),
```

Comment:
guessing we don't need the update yet?

Reply:
At this point, no.  I thought about adding it, but didn't want to incur additional unit-tests at this point.